packages/default-theme/components/atoms/SwTipPopup.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <transition name="sf-collapse-bottom">
+    <div v-if="show && isMobile" class="sw-popup container">
+      <p class="text">
+        TIP!
+        <span class="strong">Press and hold item to drag them to the cart</span>
+      </p>
+
+      <SfIcon
+        class="close"
+        :color="'#ffffff'"
+        icon="cross"
+        size="16px"
+        @click="closePopup"
+      />
+    </div>
+  </transition>
+</template>
+
+<script>
+import { SfIcon } from "@storefront-ui/vue"
+import {
```

Comment:
we don't need a mobile observer in this component

Reply:
Removed unnecessary observer